requirements/pip-tools.in

```diff
@@ -0,0 +1,13 @@
+# Just the dependencies to run pip-tools, mainly for the "upgrade" make target
+#
+# DON'T JUST ADD NEW DEPENDENCIES!!!
+#
+# If you open a pull request that adds a new dependency, you should:
+#   * verify that the dependency has a license compatible with AGPLv3
+#   * confirm that it has no system requirements beyond what we already install
+#   * run "make upgrade" to update the detailed requirements files
+
+-c constraints.txt
+
+pip-tools                 # Contains pip-compile, used to generate pip requirements files
+six                       # Pinned because pip-tools needs it, but later transifex-client says ==1.11.0
```

Comment:
I think that we can remove this dependency from here

Reply:
Removed. Now pip-tools is in pip-tools.in